### ISSUE TITLE
Impossible to update mods with names less than 4 chars.

### ISSUE THREAD

**Issue by notnotmelon** [fgardt/semantic-release-factorio#3]
Example: https://mods.factorio.com/mod/lob

I can manually update this mod
![image](https://github.com/user-attachments/assets/865556bf-c866-4ca1-88c5-0c95d0aa4f41)

However using semantic release fails with 
https://github.com/notnotmelon/lob/actions/runs/10241827847/job/28330678831

**Comment by fgardt** [comment]
The info.json validation can be skipped now. \
Just update your `.releaserc` file by replacing the line that currently reads

```json
        "semantic-release-factorio",
```

with the following

```json
        [
            "semantic-release-factorio",
            {
                "skip_validation": true
            }
        ],
```

**Comment by notnotmelon** [comment]
Thank you for the quick response,

However I am seeing errors with this fix
https://github.com/notnotmelon/lob/actions/runs/10242364786/job/28331973212

**Comment by fgardt** [comment]
small oversight, fixed in 1.6.2
